packages/unmock-core/src/persistence/failing-persistence.ts

```diff
@@ -4,15 +4,37 @@ export default class FailingPersistence implements IPersistence {
   public saveAuth() {
     throw new Error("not implemented");
   }
-  public saveToken() {
+  public saveMeta() {
+    throw new Error("not implemented");
+  }
+  public saveRequest() {
     throw new Error("not implemented");
   }
-  public saveMock() {
+  public saveResponse() {
+    throw new Error("not implemented");
+  }
+  public saveToken() {
     throw new Error("not implemented");
   }
   public loadAuth() {
     throw new Error("not implemented");
   }
+  public hasHash() {
+    throw new Error("not implemented");
+    return false;
+  }
+  public loadMeta() {
+    throw new Error("not implemented");
+    return {};
```

Comment:
`return` after `throw`?

Reply:
Ah, thought it was weird it was suddenly needed where as `loadToken` is okay. It's caused by the interface (returning undefined or a valid response).